packages/cli/generators/relation/relation.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const ArtifactGenerator = require('../../lib/artifact-generator');
+const path = require('path');
+const utils = require('../../lib/utils');
+
+module.exports = class RelationGenerator extends ArtifactGenerator {
```

Comment:
I find it confusing that we have two RelationGenerator classes, one exported by `relation/index.js` and another exported by `relation/relation.js`. How about using `BaseRelationGenerator` here?

Please update the file name accordingly, e.g. to `base-relation.generator.js`.

Reply:
Fixed. Updated class names and file names.